_data/toc.yaml

```
@@ -1392,11 +1392,15 @@ manuals:
       title: Hello Build
     - sectiontitle: Buildx
       section:
-        - path: /build/buildx/
-          title: Working with Buildx
         - path: /build/buildx/install/
           title: Install Buildx
-
+        - path: /build/buildx/
+          title: Working with Buildx
+        - path: /build/buildx/multiple-builders/
+          title: Using multiple builders
+        - path: /build/buildx/
```

The navigation is broken here:
```suggestion
        - path: /build/buildx/multiplatform-images
```